server/src/modules/maps/map-library.service.ts

```diff
@@ -0,0 +1,40 @@
+import {
+    BadRequestException,
+    Injectable,
+    NotFoundException
+} from '@nestjs/common';
+import { PrismaService } from '../repo/prisma.service';
+import {
+    Prisma
+} from '@prisma/client';
+
+
+@Injectable()
+export class MapLibraryService {
+    constructor( private prisma: PrismaService ) {}
+
+    async isMapInLibrary(userID: number, mapID: number) {
+        const mapWhere: Prisma.MapWhereInput = {
+            id: mapID
+        };
+
+        const map = await this.prisma.map.findFirst({
+            where: mapWhere
+        });
+
+        if ( !map ) {
+            throw new BadRequestException('Map does not exist');
+        }
```

Comment:
No need for curly braces around a one-liner that's just a `throw` (I generally avoid braces around one-liners everywhere). Also, since you've done `( !map )` instead of `(!map)` you've definitely not run Prettier. I'll have better guidelines on that very soon, for now can you just install the VSCode extension and set that as your formatter, then run the Format File command or whatever it's called